client/app/components/composites/Menu/Menu.css

```diff
@@ -35,7 +35,7 @@
   padding: var(--Menu_labelPaddingVertical) 0;
   color: var(--Menu_textColor);
   font-size: var(--Menu_fontSize);
-  line-height: 1.5em;
+  line-height: 1em;
```

Comment:
Line-height 1.5em aka 1.5 * 14px => 23px. 23 + 2*padding => odd number. 
It won't work well in topbar where height is even 80px.